documentation/Action-plugin-specification.md

```diff
@@ -42,6 +44,8 @@ module.exports['consoleLog'] = function (parameters: any) {
     }
   }
 }
+// src/index.ts
+export {default as action} from './consoleLog'
```

Comment:
We should probably re-write this more like a "template", with just some generic placeholder inside the `try`method, etc. I'll do this with #282.